testing/terraform/APKBUILD

```diff
@@ -2,37 +2,36 @@
 # Contributor: Gennady Feldman <gena01@gmail.com>
 # Maintainer: Thomas Boerger <thomas@webhippie.de>
 pkgname=terraform
-pkgver=0.7.12
+pkgver=0.8.1
 pkgrel=0
 pkgdesc="Building, changing, and combining infrastructure safely and efficiently"
 url="https://www.terraform.io/"
-arch="x86_64"
+arch="all"
 license="MPL 2.0"
 depends=""
-makedepends="go"
+depends_dev=""
+makedepends="$depends_dev go"
 install=""
+subpackages=""
 source="${pkgname}-${pkgver}.tar.gz::https://github.com/hashicorp/terraform/archive/v$pkgver.tar.gz"
 builddir="$srcdir/src/github.com/hashicorp/$pkgname"
 
 prepare() {
-	mkdir -p "$srcdir/src/github.com/hashicorp"
+	mkdir -p ${builddir%/*}
 	mv $srcdir/$pkgname-$pkgver "$builddir"/ || return 1
 	default_prepare
 }
 
 build() {
 	cd $builddir
-
-	GOPATH="$srcdir" go build -v -o bin/$pkgname || return 1
+	export GOPATH="$srcdir"
+	go build -v -o bin/$pkgname
 }
 
 package() {
-	cd "$builddir"
-
-	install -Dm755 bin/$pkgname \
-		"${pkgdir}/usr/bin/$pkgname" || return 1
```

Comment:
Because of https://github.com/tboerger/aports/blob/6796bae33d8bcfcf234a8ded0fd58cced8a74d7b/testing/hugo/APKBUILD#L32